test/test_embeddings.py

```diff
@@ -115,6 +115,12 @@ def test_train_batch_KL(self):
 
         self._check_results(embed)
 
+    def test_model_update(self):
+        self.embed = sample_embed()
+        self.embed.update(5)
+        self.assertEquals(self.embed.mu.shape, (5, 2))
+        self.assertEquals(self.embed.sigma.shape, (5, 1))
```

Comment:
Update function only adds new rows to the matrices. It does not do any training. We need to train separately. Which I why I am only testing for the rows being added and not the value of them.